irods/client_configuration/__init__.py

```diff
@@ -57,6 +57,16 @@ def xml_parser_default(self, str_value):
 
         return set_default_XML_by_name(str_value)
 
+    @property
+    def irods_query_limit(self):
+        import irods.query
+        return irods.query.IRODS_QUERY_LIMIT
+
+    @irods_query_limit.setter
+    def irods_query_limit(self, int_value):
+        import irods.query
+        irods.query.IRODS_QUERY_LIMIT = int(int_value)
+
 
 connections = ConnectionsProperties()
```

Comment:
Just curious if `ConnectionsProperties` is the best place for this. Would it be appropriate to introduce a `QueryProperties`?

Reply:
Maybe.

Reply:
Having a `QueryProperties` type makes the most sense to me, but it will ultimately depend on how it fits into the existing query interface.